packages/core/src/request.ts

```diff
@@ -34,7 +35,9 @@ export function eventToSentryRequest(event: Event, api: API): SentryRequest {
   if (useEnvelope) {
     const envelopeHeaders = JSON.stringify({
       event_id: event.event_id,
-      sent_at: new Date().toISOString(),
+      // We need to add * 1000 since we devide it by 1000 by default but JS works with ms precision
+      // The reason we use timestampWithMs here is that all clocks across the SDK use the same clock
```

Comment:
Rather than saying that all clocks are the same, maybe mention something about the performance API?

Reply:
I rather not, maybe we drop the usage of performance API.
We use `timestampWithMs` everywhere and we should also here.